C++/C++.sublime-syntax

```diff
@@ -578,6 +582,68 @@ contexts:
                 - include: global
             - include: expressions
 
+  module:
+    - match: '\b(export)\s+'
+      scope: meta.module.c++
+      captures:
+        1: keyword.control.module.c++
+      push:
+        - match: ''
+          set:
```

Comment:
What's the use for this extra context switch?

Reply:
Ah, just leftovers.  I had grabbed some code from the namespace block above it and probably left it in without noticing.  I'll remove it.